docs/_Docs_GetStarted.md

```diff
@@ -68,78 +73,79 @@ The final step for installation is copying over some of the files from our submo
 
-```
-yarn run test:contracts
-```
+To connect, you'll need to know the address of the colonyNetwork (which is, in reality, the address of the `etherRouter` contract; see [The Delegate Proxy Pattern](/colonynetwork/docs-the-delegate-proxy-pattern/) for more info).
 
-### Development
+`ColonyNetwork`: `0x79073fc2117dD054FCEdaCad1E7018C9CbE3ec0B`
 
-Alternatively, you can start a local test node and deploy the contracts yourself using the locally installed `truffle` package.
+You will also require Görli test ETH, and a deployed ERC20 token to import.
 
-```
-yarn run start:blockchain:client
 
-./node_modules/.bin/truffle migrate --reset --compile-all
-```
-
-For more detailed instructions, and additional steps required to set up an environment for use with [colonyJS](https://github.com/JoinColony/colonyJS), check out the colonyJS [Local Setup](/colonyjs/intro-local-setup/) documentation.
+### Access with Remix (good for experimenting)
 
-## Reputation Miner
+For simple interactions, [Remix](http://remix-alpha.ethereum.org/) is a good lightweight way to call specific functions and get network information from the contracts.
 
-The reputation mining client is usable for testing but it has limited functionality. It currently has no support for the challenge-response process to accommodate multiple submitted reputation root hashes. Still, it is possible to run a single miner instance for usable reputation scores on a local test network.
-
-### Start Mining Client
-
-You can start the reputation mining client using the following command.
+Rather than import the entire set of contracts into remix, use the included `solidity-steamroller` to flatten the needed interface contracts to the `build/flattened/` directory:
 
 ```
-node packages/reputation-miner/bin/index.js --colonyNetworkAddress { COLONYNETWORK_ADDRESS } --minerAddress { MINER_ADDRESS }
+$ yarn flatten:contracts
 ```
 
-The `minerAddress` in the execution above is the sixth account in `ganache-accounts.json` if running locally.
-
-The `colonyNetwork` address in the execution above is not the address outputted at contract deployment, but is the address of the Colony Network `EtherRouter`. See [Upgrades to the Colony Network](/colonynetwork/docs-the-delegate-proxy-pattern/) for more information about the EtherRouter design pattern.
-
-### Force Reputation Updates
+Alternatively, you can put them into the directory of your choice, e.g. `~/Downloads/`:
+```
+$ yarn steamroller /contracts/IColonyNetwork.sol > ~/Downloads/flatIColonyNetwork.sol
+```
 
-The client is set to provide a reputation update once per hour. For testing, you'll likely want to "fast-forward" your network through a few submissions to see usable reputation.
+Import the flattened contract into the remix IDE, and then call `createColony()`
```

Comment:
It's a little more than that, as they will need to create an instance of the flattened interface, then point to to an existing `EtherRouter` for the respective type, e.g. a `ColonyNetwork` representation. Only then if calling `createColony` you will be wired correctly. The above sounds to me like calling a function on an interface contract will do it.